cogs/osu/osu.py

```diff
@@ -165,6 +165,17 @@ async def bws(self,ctx,rank,bcount):
         await ctx.send("Previous Rank: **{}**    Badge Count: **{}**".format(rank,bcount))
         await ctx.send("Rank after BWS: **{}**".format(newrank))
 
+    @commands.command()
+    async def rbws(self,ctx,bws,bcount):
+        """Find the osu rank needed for a desired bws given amount of badges. -rbws [bws] [badges]"""

```

Comment:
Calculate the rank needed for your desired BWS given the amount of badges provided.